src/Entities/PaymentPlanTrait.php

```diff
@@ -43,7 +43,7 @@ public function isPayLaterOnly()
      */
     public function isPnXOnly()
     {
-        return $this->getInstallmentsCount() > 1 && (! $this->getDeferredDays() && ! $this->getDeferredMonths());
+        return $this->getInstallmentsCount() >= 1 && (! $this->getDeferredDays() && ! $this->getDeferredMonths());
```

Comment:
@Francois-Gomis , I am not ok with that modification for 2 reasons:
1. this is a Pull request for auth validation, not for p1x
2. you should add a `isPayNow()` method if you want test p1x (I think we should assume that in pnx, n is always `>=2`)